skycoin/skywire
# Switch systray repo

Did you run `make format && make check`? Yes

Fixes #1165 #1167

 Changes:	
- use our fork of fyne-io fork, actually skycoin/systray
- improve vpn status of systray

How to test this PR:
1. #1165
    - `make build-systray`
    - run in other machine that doesn't have systray dependencies. 
2. #1167 
    - do changes on 1167 bug report and run again (both with correct and wrong dsmgd server address), no race condition should happened.
